ansible_collections/arista/avd/docs/release-notes/3.x.x.md

```diff
@@ -20,9 +20,136 @@ See the release notes and role documentation for all new additions.
+
+### New Features and enhancements
+
+#### New features and enhancements in eos_cli_config_gen
+
+- 9f8d26898 Feat(eos_cli_config_gen): Support various ip igmp snooping commands (#1464)
+- 464ae572e Feat(eos_cli_config_gen): Support setting OSPF distance (#1438)
+- 7b329bd1c Feat(eos_cli_config_gen): Support for Flex Encap and l2dot1q subinterfaces (#1452)
+- ec7c8c5d1 Feat(eos_cli_config_gen): Support for DHCP relay (#1439)
+- b5a3f0da4 Feat(eos_cli_config_gen): Support EVPN ethernet segment options on ethernet-interfaces (#1437)
+- 685169963 Feat(eos_cli_config_gen): Support disable commands of spanning-tree bpduguard and bpdufilter (#1375)
+- 5ace05a47 Feat(eos_cli_config_gen): Support cos in policy-map and service-policy in qos profile (#1504)
+- e117eda27 Feat(eos_cli_config_gen): port-channel-interfaces enhancement (#1396)
+- de422ac54 Feat(eos_cli_config_gen): logging facility (#1490)
+- 95ed0568e Feat(eos_cli_config_gen): IPv6 ACL support for QOS policy map class maps (#1515)
+- 48ecb7011 Feat(eos_cli_config_gen): Improve data model for extended IP ACLs #1411 (#1424)
+- 86c1f8682 Feat(eos_cli_config_gen): Extend platform sand and add documentation (#1425)
+- 23a0d4abf Feat(eos_cli_config_gen, eos_designs): Support bfd in router_bgp vrf neighbors (#1445)
+- 44b957883 Feat(eos_cli_config_gen): default_originate for router_bgp neighbors (#1380)
+- 1c71d6987 Feat(eos_cli_config_gen): Add vpws control-word, mtu and label flow knobs (#1509)
+- 2493a992c Feat(eos_cli_config_gen): add support to shutdown individual VRF neighbors in bgp (#1476)
+- 335a59e4d Feat(eos_cli_config_gen): Add support for Vxlan DSCP mapping (#1462)
+- 9a1be3cbd Feat(eos_cli_config_gen): Add support for TACACS AES-256-GCM keys (#1456) (#1466)
+- 9a8c8378c Feat(eos_cli_config_gen): Add support for segment-routing traffic-engineering (#1486)
+- db0d25005 Feat(eos_cli_config_gen): Add support for reversible encryption algorithm #1468 (#1469)
+- f21e5dbad Feat(eos_cli_config_gen): Add support for OSPF Distribute List (#1442)
+- 536bc64ed Feat(eos_cli_config_gen): Add support for MPLS and LDP on port-channel interfaces (#1487)
+- 1fce085d9 Feat(eos_cli_config_gen): Add support for as-path filter (#1429)
+- e7b0d339e Feat(eos_cli_config_gen): Add spanning_tree_guard to switched ethernet interfaces (#1460)
+- a6b62e33f Feat(eos_cli_config_gen): Add PIM RP addresses on VRF instance (#1484)
+- 6b69e6405 Feat(eos_cli_config_gen): Add Monitor Session support (#1435)
+- 6566afd12 Feat(eos_cli_config_gen): Add MAC ACL support to Ethernet and Port-channels (#1422)
+- 3f0dae75c Feat(eos_cli_config_gen): Add lldp tlv transmit ztp vlan (#1458)
+- 30a40780a Feat(eos_cli_config_gen): Add ISIS authentication and hello padding (#1453)
+- a7c3b49d9 Feat(eos_cli_config_gen): Add fast-failover and router-id support under router general (#1495)
+- 52e981423 Feat(eos_cli_config_gen): Added support for PFC under ethernet_interf… (#1398)
```

Comment:
Again I found the weird unknown character type because of the last 3 dots. Is it ok to manually edit this one?